src/components/All-issues/AllIssues.tsx

```diff
@@ -0,0 +1,238 @@
+"use-client"
+import style from "./index.module.css"
+import style2 from "../main-content/general.module.css"
+import { useTheme } from "../contextApi/ThemeContext"
+import Image from "next/image"
+import * as Dialog from "@radix-ui/react-dialog"
+import { DiscIcon } from "@radix-ui/react-icons"
+import { useState } from "react"
+
+interface Labels2Element {
+    language: string
+    number: number
+    chat: {
+        chat_number: number
+        chat_icons: string
+    }
+    last_updated: string
+    icon: string
+}
+
+interface Data {
+    id: number
+    header: string
+    company: { smallIcon: string; name: string }
+    behaviour_text: string
+    expected_behaviour_text: string
+    labels: Array<string>
+    labels2: Labels2Element[]
```

Comment:
what is labels2? this type name is not descriptive

Reply:
Noted